<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSubtractTestPage</class>
 <widget class="QWidget" name="AddSubtractTestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStartTest">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>测试开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNStopTest">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>510</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>测试结束</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_testStatus">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>517</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试进行中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>列车速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>手柄级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroSpeed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0.8km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_level">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>+60%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0-40km/h
加速度测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0-80km/h
加速度测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>395</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>减速度测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>平均加速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_40Speed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0.8m/s^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_40Level">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>+60%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>平均加速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_80Speed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0.8m/s^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_80Level">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>+60%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>制动初速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>平均减速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_subBrakeSpeed">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>30km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_subSpeed">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>-0.8m/s^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>制动距离</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brakeDistance">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>40m</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>制动模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brakeMode">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>电制动/空气制动</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_subLevel">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>-80%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
